BedrockServer.h

```diff
@@ -206,10 +206,18 @@ class BedrockServer : public SQLiteServer {
     // When a peer node logs in, we'll send it our crash command list.
     void onNodeLogin(SQLiteNode::Peer* peer);
 
-    // Control the command port. The server will toggle this as necessary, unless manualOverride is set,
-    // in which case the `suppress` setting will be forced.
+    // You must block and unblock the command port with *identical strings*.
+    void blockCommandPort(const string& reason);
+    void unblockCommandPort(const string& reason);
+
+    // Legacy version of above.
     void suppressCommandPort(const string& reason, bool suppress, bool manualOverride = false);
 
+    // Reasons for each request so close the command port mapped to the instance of commandPortSuppressionCount that
```

Comment:
```suggestion
    // Reasons for each request to close the command port mapped to the instance of commandPortSuppressionCount that
```
?

(also, the line return on the line below seems unnecessary)